features/named-color.yml

```
@@ -0,0 +1,9 @@
+name: Named colors
+description: A standard set of colors that can be referenced by names, such as `red` or `limegreen`.
```

These are generally formed as complete sentences. And this doesn't seem to mention anywhere that we're talking about CSS values.

Very optionally, an opportunity to use an also known as, if we want to reference some ancient history. (This is me mostly demonstrating a convention to you than earnestly advocating for it.)

```suggestion
description: Some CSS color values can be referenced by name, such as `red` or `limegreen`. They stand in for specific RGB color values. Also known as the X11 color set.
```

---

I'll omit the X11 reference as there are a few minor differences. Although while looking into X11, I finally found the reason why `gray` is darker than `dark gray` 😆